### ISSUE TITLE
if user has multiple fb tabs open, only open fbmood after last one is closed

### ISSUE THREAD

**Issue by ehmorris** [ehmorris/Facebook-Mood#6]

**Comment by nixpulvis** [comment]
The main problem with this is that there is a decently large population of people that literally never "exit" facebook, unless they quit chrome. This is going to be a problem since we can't capture the users quitting the application.

**Comment by ehmorris** [comment]
true. see comments in https://github.com/ehmorris/fbmood/issues/10
